### PR TITLE
update policy support for attendance report

### PR DESCRIPTION
**Pull request by changqingren**
update policy support for attendance report using the latest values.

Currently the config for the attendance report has changed. update the document using the latest version:
image
image

### CODE REVIEW

**Comment by changqingren** [comment]
@microsoft-github-policy-service agree company="Microsoft"

**Comment by changqingren** [comment]
Currently the config for the attendance report has changed. update the document using the latest version:
image
image

**Comment by changqingren** [comment]
@mkbond007 @JohanFreelancer9  Could you please help to review the PR. Thanks!

File: Teams/teams-analytics-and-reports/meeting-attendance-report.md [hunk 1]
```diff
@@ -35,15 +35,15 @@ For education tenants, the attendance report can be used to track student attend
...
-    - **Everyone** - The attendance report is available for all webinars, town halls, or meetings organizers create; organizers can't turn off attendance reports.
-
-1. For **Who is in the report**, choose one of the following options:
-    - **Everyone, but participants can opt-out** -  This option is the **default setting.** The attendance report initially includes all participants. To opt out, participants can set the **Identify me in attendance reports** toggle to **off** in their Teams privacy settings.
-    - **No one, but participants can opt-in** - The attendance report initially excludes all participants. To opt in, Participants can set the **Identify me in attendance reports** toggle to on or off in their Teams privacy settings.
-    - **Everyone** - The attendance report includes all participants, and participants can't opt out.
-    - **No one** - The attendance report excludes all participants, and participants can't opt in.
+    - **On , but organizers can turn it off** - Organizers control whether attendance reports are on or off for their meetings, town halls or webinars.
```
Comment (JohanFreelancer9) [inline Teams/teams-analytics-and-reports/meeting-attendance-report.md:38]:
```suggestion
    - **On, but organizers can turn it off** - Organizers control whether attendance reports are on or off for their meetings, town halls, or webinars.
```

File: Teams/teams-analytics-and-reports/meeting-attendance-report.md [hunk 2]
```diff
@@ -35,15 +35,15 @@ For education tenants, the attendance report can be used to track student attend
...
+    - **Off** - Organizers can't view or download attendance reports for webinars, town halls, or meetings they've organized.
+    - **On** - The attendance report is available for all webinars, town halls, or meetings organizers create; organizers can't turn off attendance reports.
+
+1. For **Include attendees in the report**, choose one of the following options:
+    - **Yes, but allow opt out** -  This option is the **default setting.** The attendance report initially includes the participants who have been assigned this policy value. To opt out, participants can set the **Identify me in attendance reports** toggle to **off** in their Teams privacy settings.
+    - **No, but allow opt in** - The attendance report initially excludes the participants who have been assigned this policy value. To opt in, Participants can set the **Identify me in attendance reports** toggle to on or off in their Teams privacy settings.
```
Comment (JohanFreelancer9) [inline Teams/teams-analytics-and-reports/meeting-attendance-report.md:44]:
```suggestion
    - **No, but allow opt in** - The attendance report initially excludes the participants who have been assigned this policy value. To opt in, participants can set the **Identify me in attendance reports** toggle to on or off in their Teams privacy settings.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @changqingren, kindly commit the suggested changes. Many thanks.

**Review by mkbond007 — APPROVED** [review]

**Pull request merged**